config/routes.rb

```diff
@@ -8,6 +8,9 @@
   get "/eventschool", to: "pages#eventschool", via: :all
   get "/event", to: "pages#event", via: :all
   get "/event/register/1", to: "pages#eventregistration", via: :all
+  get "/event/register/2", to: "pages#eventregistration2", via: :all
```

Comment:
This could be changed to a single route/action:

```
get "/event/register/:step_number", to: "pages#eventregistration", via: all
```
You could then do something like this in the action:

```
def eventregistration
  render template: "pages/events/registration/step#{params[:step_number]}"
end
```

Reply:
i needed this info, but couldnt find it :)